addons/arrays/test_shuffle.sqf

```diff
@@ -21,18 +21,14 @@ TEST_OP(count _result,==,count _original,_fn);
     TEST_OP(_x,in,_original,_fn);
 } forEach _result;
 
-// Test depecated version.
+// test modify original
```

Comment:
That aswell - use of the "direct array" parameter form is discouraged, but the function still works. Why break it?